lib/structs/events/account_data/ignored_users.cpp

```diff
@@ -0,0 +1,33 @@
+#include <nlohmann/json.hpp>
+#include <unordered_map>
+
+#include "mtx/events/account_data/ignored_users.hpp"
+
+namespace mtx {
+namespace events {
+namespace account_data {
+
+void
+from_json(const nlohmann::json &obj, IgnoredUsers &content)
+{
+    const auto map = obj.at("ignored_users").get<std::unordered_map<std::string, nlohmann::json>>();
```

Comment:
You should add a guard here, that the key actually exists. Some clients might remove it, if it is empty :)